cfme/storage/volume.py

```diff
@@ -69,9 +69,8 @@ class VolumeAllView(VolumeView):
 
     @property
     def is_displayed(self):
-        return (
-            self.in_volume and
-            self.entities.title.text == 'Cloud Volumes'
+        return (self.in_volume and
```

Comment:
I don't see any change to the content in any of these, just formatting. The old formatting is preferred, and is what `Black` will enforce when its enabled for this directory.